internal/cmd/io_binary.go

```diff
@@ -105,6 +115,34 @@ func NewBinaryIO(ctx context.Context, id string, uri *url.URL) (_ UpstreamIO, er
 	}, nil
 }
 
+// sanitizePath parses the URL object and returns an absolute path to the logging driver
+func sanitizePath(uri *url.URL) (sanitized string) {
+	path := uri.Path
+	// Convert windows-style slashes to linux-style to clean the path
+	path = strings.Replace(path, "\\", "/", -1)
```

Comment:
It looks like we could simplify this by just doing the following:

```
path = strings.TrimPrefix(path, "/")
path = filepath.Clean(path)
```

This seems to result in a proper Windows path that can be used with e.g. `os/exec.Command`.

Reply:
oh yeah, this is definitely simpler 😄